src/test/java/nurseybook/logic/commands/DeleteTagCommandTest.java

```diff
@@ -1,32 +1,31 @@
-package seedu.address.logic.commands;
-
+package nurseybook.logic.commands;
+
+import static nurseybook.commons.core.Messages.MESSAGE_INVALID_ELDERLY_DISPLAYED_INDEX;
+import static nurseybook.logic.commands.CommandTestUtil.SET_ONE_TAG;
+import static nurseybook.logic.commands.CommandTestUtil.SET_TWO_TAGS;
+import static nurseybook.logic.commands.CommandTestUtil.VALID_TAG_DIABETES;
+import static nurseybook.logic.commands.CommandTestUtil.VALID_TAG_FRIEND;
+import static nurseybook.logic.commands.CommandTestUtil.assertCommandFailure;
+import static nurseybook.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static nurseybook.testutil.TypicalIndexes.INDEX_FIRST;
+import static nurseybook.testutil.TypicalIndexes.INDEX_SECOND;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static seedu.address.logic.commands.CommandTestUtil.SET_ONE_TAG;
-import static seedu.address.logic.commands.CommandTestUtil.SET_TWO_TAGS;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_DIABETES;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_FRIEND;
-import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
-import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
-import static seedu.address.logic.commands.CommandTestUtil.showElderlyAtIndex;
-import static seedu.address.testutil.TypicalElderlies.getTypicalAddressBook;
```

Comment:
Missing imports in this file as well that would be better to include at the start!